.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,21 @@
+name: Node.js CI
```

Comment:
can we please keep this in travis and migrate from travis to github actions in a later pr?